src/net.cpp

```diff
@@ -1156,6 +1156,30 @@ void CConnman::AcceptConnection(const ListenSocket& hListenSocket) {
     RandAddEvent((uint32_t)id);
 }
 
+bool CConnman::AddConnection(const std::string& address, const ConnectionType conn_type)
```

Comment:
0c4d673716f2eb37efdb81ad2d7d20f69de960b3 nit, pass by value or by reference to const, but not const value, idem for the declaration in net.h
```suggestion
bool CConnman::AddConnection(const std::string& address, ConnectionType conn_type)
```

Reply:
done